examples/nD_points.py

```diff
@@ -32,5 +32,5 @@
         ]
     )
     viewer.add_points(
-        points, size=[0, 6, 10, 10], face_color='blue', n_dimensional=True
+        points, anisotropy=[0, 0.5, 1, 1], size=10, face_color='blue', n_dimensional=True
```

Comment:
(a) I would put size before anisotropy, because it is the more intuitive and universal argument, in that most people will deal with size but not anisotropy; and
(b) I strongly suggest renaming anisotropy to scale, to make it consistent with the image layer. Anisotropy is not a name that most people will find intuitive!